### PR TITLE
NOTIF-361 Protect Event log with new RBAC permission

### PR DESCRIPTION
**Pull request by bennetelli**

### CODE REVIEW

File: backend/src/main/java/com/redhat/cloud/notifications/auth/rbac/RbacIdentityProvider.java [hunk 1]
```diff
@@ -108,6 +110,9 @@
                                                 })
                                                 // Otherwise, we can finish building the QuarkusSecurityIdentity and return the result
                                                 .onItem().transform(rbacRaw -> {
+                                                    if (rbacRaw.canRead("events", "events")) {
```
Comment (josejulio) [inline backend/src/main/java/com/redhat/cloud/notifications/auth/rbac/RbacIdentityProvider.java:113]:
```suggestion
                                                    if (rbacRaw.canRead("notifications", "events")) {
```
Can't really remember the order, but one is the "application" and other is the "resource"

File: backend/src/test/resources/rbac-examples/rbac_example_full_access.json [hunk 2]
```diff
@@ -15,6 +15,10 @@
       "permission": "notifications:*:*",
       "resourceDefinitions": []
     },
+    {
+      "permission": "events:*:*",
```
Comment (josejulio) [inline backend/src/test/resources/rbac-examples/rbac_example_full_access.json:19]: We probably don't need this one, the first is the application (notifications). We are not creating a new `application`, only a new `resource`

Comment (bennetelli) [reply backend/src/test/resources/rbac-examples/rbac_example_full_access.json:19]: i just tested it. When I remove this, the EventServiceTest.testAllQueryParams() fails.

Comment (josejulio) [reply backend/src/test/resources/rbac-examples/rbac_example_full_access.json:19]: It goes with the other comment :)

File: backend/src/main/java/com/redhat/cloud/notifications/auth/rbac/RbacIdentityProvider.java (hunk 1 above)
Comment (bennetelli) [reply backend/src/main/java/com/redhat/cloud/notifications/auth/rbac/RbacIdentityProvider.java:113]: The first one is the application. The second one is an item.. maybe the parameter has got a misleading name.. probably this is the resource.

How should this be changed?

Comment (josejulio) [reply backend/src/main/java/com/redhat/cloud/notifications/auth/rbac/RbacIdentityProvider.java:113]: Item should be the resource (or I confused the RBAC lingo). So my suggestion looks OK

**Comment by josejulio** [comment]
I think this one should stay as draft, unless the permission already exists.

**Comment by bennetelli** [comment]
> I think this one should stay as draft, unless the permission already exists.

see: https://github.com/RedHatInsights/rbac-config/pull/217

File: backend/src/test/resources/rbac-examples/rbac_example_full_access.json (hunk 2 above)
Comment (bennetelli) [reply backend/src/test/resources/rbac-examples/rbac_example_full_access.json:19]: ok, then it's resolved I guess :)

**Pull request closed without merging**